docs/user/aws/tested_instance_types_aarch64.md

```diff
@@ -0,0 +1,2 @@
+* `c6g.*`
+* `m6i.*`
```

Comment:
while reviewing https://github.com/openshift/openshift-docs/pull/41796, looks like it should be `m6g.*`, cc @aleskandro

Reply:
docs/user/azure/tested_instance_types.md need be updated , now I'm waiting for the test result.

Reply:
@yunjiang29 yes, it should